index.html

```diff
@@ -222,5 +222,37 @@ <h2>Skills</h2>
             </div>
         </div>
     </section>
+    <section class="contact-form" id="contact">
+        <div class="contact-text">
+            <p>
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </p>
+        </div>
+        <div class="contact-area">
+            <form action="https://formspree.io/f/xvolllzo" method="post" id="frmContact">
+              <label for="name">
+                <input type="text" name="name" id="name" maxlength="30" class="contact-input" placeholder="Enter your names..."
+                required/>
+              </label>
+              <label for="email">
+                <input type="email" name="email" id="email" class="contact-input" placeholder="Enter your gmail..."
+                required/>
+              </label>
+              <label for="text">
+                <textarea name="text" id="text" maxlength="500" class="contact-input" placeholder="Enter text here" cols="23" rows="5" required></textarea>
+              </label>
+                <input id="btn" type="submit" class="submit-btn dark-font" value="Get in Touch" />
```

Comment:
- [ ] As stated before by the previous reviewer, kindly make sure that you implement the button states eg on hover, on submitting to make your design according to Figma guidelines.
> states
![image](https://user-images.githubusercontent.com/53402600/177527459-4ce1c991-241d-41f7-b1e0-e2bf0261c1b5.png)

Reply:
Please specify which buttons

Reply:
Should I apply it to all buttons on my page or??